audioipc/src/rpccore.rs

```diff
@@ -153,32 +149,64 @@ const RPC_CLIENT_INITIAL_PROXIES: usize = 32; // Initial proxy pre-allocation pe
 #[derive(Debug)]
 struct ProxyManager<Response> {
     proxies: Mutex<Slab<Sender<Response>>>,
+    connected: AtomicBool,
```

Comment:
How about using ` Mutex<Option<Slab<Sender<Response>>>>` for `proxies` that has `Mutex::new(Some(Slab::with_capacity(RPC_CLIENT_INITIAL_PROXIES)))` initially and becomes `None` after `disconnect_handler`?

Reply:
That's a much better idea, done.  Thanks!